<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>detailsWindow</class>
 <widget class="QMainWindow" name="detailsWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>500</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Details</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/logo/logo.png</normaloff>:/logo/logo.png</iconset>
  </property>
  <property name="iconSize">
   <size>
    <width>50</width>
    <height>50</height>
   </size>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="verticaBlFrame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>500</width>
      <height>500</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>500</width>
      <height>500</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>500</width>
      <height>500</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(205,214,219);
border: 1px solid rgb(205,214,219);</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item alignment="Qt::AlignHCenter">
      <widget class="QLabel" name="referencelImageLabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border: 2px solid rgb(85,101,85);
border-radius: 10px;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="resources.qrc">:/assets/assets/006-virus.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item alignment="Qt::AlignHCenter">
      <widget class="QFrame" name="horizontalFrame_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>490</width>
         <height>70</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>490</width>
         <height>70</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border-radius: 6px;</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>50</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>50</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(255,255,255);
background-color: rgb(85,101,85);
font: 10pt &quot;Product Sans Medium&quot;;
border: 4px solid rgb(85,101,85);
border-radius: 5px; 
</string>
          </property>
          <property name="text">
           <string>Name</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="diseaseNameLabel">
          <property name="minimumSize">
           <size>
            <width>364</width>
            <height>50</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>364</width>
            <height>50</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Product Sans Medium</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(205,214,219);
border: 1px solid  rgb(85,101,85);
padding-left : 5px;</string>
          </property>
          <property name="text">
           <string>NULL</string>
          </property>
          <property name="textInteractionFlags">
           <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item alignment="Qt::AlignHCenter">
      <widget class="QFrame" name="horizontalFrame_4">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>490</width>
         <height>70</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>490</width>
         <height>70</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border-radius: 6px;</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>50</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>50</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(255,255,255);
background-color: rgb(85,101,85);
font: 10pt &quot;Product Sans Medium&quot;;
border: 4px solid rgb(85,101,85);
border-radius: 5px; 
</string>
          </property>
          <property name="text">
           <string>Pathogen</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="pathogenLabel">
          <property name="minimumSize">
           <size>
            <width>364</width>
            <height>50</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>364</width>
            <height>50</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Product Sans Medium</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(205,214,219);
border: 1px solid  rgb(85,101,85);
padding-left : 5px;</string>
          </property>
          <property name="text">
           <string>NULL</string>
          </property>
          <property name="textInteractionFlags">
           <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item alignment="Qt::AlignHCenter">
      <widget class="QFrame" name="horizontalFrame_3">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>490</width>
         <height>90</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>490</width>
         <height>90</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border-radius: 6px;</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>80</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>80</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(255,255,255);
background-color: rgb(85,101,85);
font: 10pt &quot;Product Sans Medium&quot;;
border: 4px solid rgb(85,101,85);
border-radius: 5px; 
</string>
          </property>
          <property name="text">
           <string>Symptoms</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QScrollLabel" name="symptomsLabel">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>364</width>
            <height>80</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>364</width>
            <height>80</height>
           </size>
          </property>
          <property name="baseSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Product Sans Medium</family>
            <pointsize>10</pointsize>
            <weight>50</weight>
            <italic>false</italic>
            <bold>false</bold>
           </font>
          </property>
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="styleSheet">
           <string notr="true">QScrollArea{
	background-color: rgb(205,214,219);
	border: 1px solid  rgb(85,101,85);
	padding-left: 5px;
	font: 10pt &quot;Product Sans Medium&quot;;
	padding-top: 2px;
}


</string>
          </property>
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>352</width>
             <height>68</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">border: none;
border-radius: 0px;</string>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item alignment="Qt::AlignHCenter">
      <widget class="QFrame" name="horizontalFrame_5">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>490</width>
         <height>90</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>490</width>
         <height>90</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border-radius: 6px;</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QLabel" name="label_6">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>80</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>80</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(255,255,255);
background-color: rgb(85,101,85);
font: 10pt &quot;Product Sans Medium&quot;;
border: 4px solid rgb(85,101,85);
border-radius: 5px; 
</string>
          </property>
          <property name="text">
           <string>Favourable Conditions</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QScrollLabel" name="favCondLabel">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>364</width>
            <height>80</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>364</width>
            <height>80</height>
           </size>
          </property>
          <property name="baseSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Product Sans Medium</family>
            <pointsize>10</pointsize>
            <weight>50</weight>
            <italic>false</italic>
            <bold>false</bold>
           </font>
          </property>
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="styleSheet">
           <string notr="true">QScrollArea{
	background-color: rgb(205,214,219);
	border: 1px solid  rgb(85,101,85);
	padding-left: 5px;
	font: 10pt &quot;Product Sans Medium&quot;;
	padding-top: 2px;
}


</string>
          </property>
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents_2">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>352</width>
             <height>68</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">border: none;
border-radius: 0px;</string>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QScrollLabel</class>
   <extends>QScrollArea</extends>
   <header>qscrolllabel.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
